### ISSUE TITLE
How to manage partial configs on a pull server?

### ISSUE THREAD

**Issue by TheRealBenForce** [PowerShell/xPSDesiredStateConfiguration#313]
I have a question that I haven't found any good blog posts addressing.

I have a security baseline configuration I would like applied to all servers in my environment. This gets pulled from the pull server at build time. I also have the pull server configuration that gets pushed to itself. The pull server obviously needs both. What is the most graceful way to handle this? Would it be establishing the pull server's LCM by pushing the configuration that makes it a pull server, then re-establishing the LCM with partial configurations, one being push, and one being pull? 

[Partial configurations in mixed push and pull modes](https://msdn.microsoft.com/en-us/powershell/dsc/partialconfigs)

**Comment by BerheAbrha** [comment]
@TheRealBenForce,  one way to do this is to define the local configuration of the pull server as having  two partials configuration.  Something like the following ...
`
[DscLocalConfigurationManager()]
configuration pullServerMetaConfig
{
  Settings {
  RefreshMode =  "PUSH"
  }
  
  #Security configuration 
  PartialConfiguration BaseConfiguration
  {
    
  }
  #pullServer configuration
  PartialConfiguration PullServerConfiguration
  {
  }
}
pullServerMetaConfig
slcm -path pullServerMetaConfig -Verbose`

**Comment by TheRealBenForce** [comment]
What I ended up doing was converting the PullServer from push to pull from itself after it was already established as a pull server. This was the easiest way for me to implement the security controls into my existing code base. I tested by wiping out the DSC website on the Pull Server and it got recreated. Is there a compelling reason _not_ to do this? Also this is implemented on EC2. At some point I'll be implementing health checks where if there is an issue with the pull server, it can wipe itself out and rebuild.
